internal/config/cassandra_test.go

```diff
@@ -85,115 +87,253 @@ func TestCassandra_Bind(t *testing.T) {
+							"cassandra-2.cassandra.default.svc.cluster.local",
+						},
+						CQLVersion:        "3.0.0",
+						ProtoVersion:      0,
+						Timeout:           "600ms",
+						ConnectTimeout:    "3s",
+						Port:              9042,
+						Keyspace:          "vald",
+						NumConns:          2,
+						Consistency:       "quorum",
+						SerialConsistency: "localserial",
+						Username:          "root",
+						Password:          "password",
+						PoolConfig: &PoolConfig{
+							DataCenter:               "",
+							DCAwareRouting:           false,
+							NonLocalReplicasFallback: false,
+							ShuffleReplicas:          false,
+							TokenAwareHostPolicy:     false,
+						},
+						RetryPolicy: &RetryPolicy{
+							NumRetries:  3,
+							MinDuration: "10ms",
+							MaxDuration: "1s",
+						},
+						ReconnectionPolicy: &ReconnectionPolicy{
+							MaxRetries:      3,
+							InitialInterval: "100ms",
+						},
+						HostFilter: &HostFilter{
+							Enabled:    false,
+							DataCenter: "",
+							WhiteList:  []string{},
+						},
+						SocketKeepalive:   "0s",
+						MaxPreparedStmts:  1000,
+						MaxRoutingKeyInfo: 1000,
+						PageSize:          5000,
+						TLS: &TLS{
+							Enabled:            false,
+							Cert:               "/path/ro/cert",
+							Key:                "/path/to/key",
+							CA:                 "/path/to/ca",
+							InsecureSkipVerify: false,
+						},
+						Net: &Net{
+							DNS: &DNS{
+								CacheEnabled:    true,
+								RefreshDuration: "5m",
+								CacheExpiration: "24h",
+							},
+							Dialer: &Dialer{
+								Timeout:          "30s",
+								KeepAlive:        "10m",
+								DualStackEnabled: false,
+							},
+							TLS: &TLS{
+								Enabled:            false,
+								Cert:               "/path/ro/cert",
+								Key:                "/path/to/key",
+								CA:                 "/path/to/ca",
+								InsecureSkipVerify: false,
+							},
+							SocketOption: &SocketOption{
+								ReusePort:                true,
+								ReuseAddr:                true,
+								TCPFastOpen:              true,
+								TCPCork:                  false,
+								TCPDeferAccept:           true,
+								IPTransparent:            false,
+								IPRecoverDestinationAddr: false,
+							},
+						},
+						EnableHostVerification:   false,
+						DefaultTimestamp:         true,
+						ReconnectInterval:        "",
+						MaxWaitSchemaAgreement:   "",
+						IgnorePeerAddr:           false,
+						DisableInitialHostLookup: false,
+						DisableNodeStatusEvents:  false,
+						DisableTopologyEvents:    false,
+						DisableSchemaEvents:      false,
+						DisableSkipMetadata:      false,
+						DefaultIdempotence:       false,
+						WriteCoalesceWaitTime:    "200ms",
+						KVTable:                  "kv",
+						VKTable:                  "vk",
+						VectorBackupTable:        "backup_vector",
+					},
+				},
+			}
+		}(),
+		func() test {
+			key := "PASSWORD"
+			val := "cassandra_password"
```

Comment:
Do you need other keys as well?:thinking:

Reply:
🤔  
it is not a must, only depends what you think